### ISSUE TITLE
Unable to control HttpStream volume

### ISSUE THREAD

**Issue by jptrmn** [CDrummond/cantata#1725]
Hi, 
I am unable to hear HTTP Stream output from cantata. 

When I try to manipulate the volume slider in the GUI, the slider stays at about 50 percent. 

The debug log produces the following output:

```
$ cantata --debug=http-stream
2021-07-02 18:02:09 - HttpStream streamUrl "http://xxx.xxx.xxx.xxx:8000/mpd.flac"
2021-07-02 18:02:09 - defaultServiceProvider::requestService(): no service found for - "org.qt-project.qt.mediaplayer"
2021-07-02 18:02:09 - HttpStream setVolume 0
2021-07-02 18:02:09 - HttpStream volume 0
2021-07-02 18:02:09 - HttpStream updateStatus 1 65535
2021-07-02 18:02:09 - HttpStream volume 0
2021-07-02 18:02:09 - HttpStream updateStatus 1 1
2021-07-02 18:02:09 - HttpStream updateStatus 1 1
2021-07-02 18:02:14 - HttpStream updateStatus 1 1
2021-07-02 18:02:17 - HttpStream setVolume 62
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 51
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 53
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 55
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 59
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 63
2021-07-02 18:02:17 - HttpStream volume 0
2021-07-02 18:02:17 - HttpStream setVolume 64
2021-07-02 18:02:17 - HttpStream volume 0
...
```
I have checked that MPD does produce output at `http://xxx.xxx.xxx.xxx:8000/mpd.flac` with the browser and I have also tried with mp3 (`...:8123/mpd.mp3`)

I am using Cantata `2.4.1` from the repos of Ubuntu 20.04; MPD Protocol: `0.21.4`

Any ideas as of why the volume stays at zero?

Thanks and Best Regards,
Josef

**Comment by CDrummond** [comment]
Sorry, no idea what's wrong on your system/build. It's not a feature I implemented (was another user), and not something I've ever used - or really liked (due to the delay). However, I have just on a build from maste and it does work.

Again, sory but I have no time to look into issues that I cannot recreate. I stopped using MPD, and Cantata, well over 3 years ago. Cantata is in a bug fix only state, and for me, this is working so is not a bug.
